src/Service/Lambda/CHANGELOG.md

```diff
@@ -2,18 +2,15 @@
 
 ## NOT RELEASED
 
+## 0.5.0
+
 ### Deprecation
 
 - Removes methods `getServiceCode`, `getSignatureVersion` and `getSignatureScopeName` from Client.
 
 ### Fixed
 
 - Fixed issue when Layer, Function or Version contained a special char `#`
-
-## 0.4.2
```

Comment:
I think it is better to use "NOT RELEASED" instead of prematurely add a version number.

Reply:
Agreed. That's why I added `NOT RELEASED` on all CHANGLOG files